context_interp/vgg19_features/vgg19_features.py

```diff
@@ -0,0 +1,25 @@
+import tensorflow as tf
+from context_interp.vgg19_features.model.model import Vgg19
+
+
+class Vgg19Features:
+
+    def load_pretrained_weights(self):
+        vgg19_data = Vgg19.load_params_dict(load_small=True)
+        self.vgg19 = Vgg19(data_dict=vgg19_data)
+
+    def get_context_features(self, images):
+        """
+        :param images: A Tensor. Of shape [batch, H, W, num_features].
+        :return: A Tensor. Of shape [batch, H, W, 64].
+        """
+        image_features, _ = self.vgg19.build_up_to_conv1_2(images, trainable=False)
```

Comment:
Huh, this only uses one feature map? Why not have the option to use more? Same question for the perceptual features.

Reply:
Do you want it to return all the layers of VGG that are of the given resolution? The only reason for not doing more is simply that it wasn't being done in the paper, and I wasn't sure what we'd want to do more of.

Reply:
Experiments on that should be done later so maybe add a TODO on this.

Reply:
This is not something that's as concrete as other TODO tasks, it's more like an optional thing to try -- we will have many more of these things soon, and I want to keep tabs on these elsewhere (perhaps in the wiki?).